disnake/user.py

```diff
@@ -162,12 +166,27 @@ def banner(self) -> Optional[Asset]:
         .. versionadded:: 2.0
 
         .. note::
+
             This information is only available via :meth:`Client.fetch_user`.
         """
         if self._banner is None:
             return None
         return Asset._from_banner(self._state, self.id, self._banner)
 
+    @property
+    def avatar_decoration(self) -> Optional[Asset]:
+        """Optional[:class:`Asset`]: Returns the user's avatar decoration asset, if available.
```

Comment:
Should note that this always returns an (animated) png, and that `Asset.with_size` and `Asset.with_format` don't work.

Reply:
dcc3dd4c2de146398673910c9359e0409cecd736 ~~(I hope this wording is fine 😩 )~~